app/controllers/merchant/items_controller.rb

```diff
@@ -1,7 +1,77 @@
-class Merchant::ItemsController < ApplicationController
+class Merchant::ItemsController < Merchant::BaseController
+  def index
+    @merchant = Merchant.find(current_user.merchant_id)
+    @items = @merchant.items
+    render 'items/index'
+  end
+
+  def new
+    @merchant = Merchant.find(current_user.merchant_id)
+    @item = Item.new
+  end
+
+  def create
+    @merchant = Merchant.find(current_user.merchant_id)
```

Comment:
I think `Merchant.find(current_user.merchant_id)` can just be `current_user` as that should return a merchant object. We can add a guard class of `if current_merchant?` if we want, but I don't think that is necessary as the admin path will be different.

Reply:
current user will return the ID for the user, not the attached merchant ID which is what I need to set the merchant.